debugging/book-library/script.js

```diff
@@ -1,103 +1,68 @@
 let myLibrary = [];
 
-window.addEventListener("load", function (e) {
-  populateStorage();
+window.addEventListener("load", () => {
+  if (myLibrary.length === 0) {
+    myLibrary.push(new Book("Robinson Crusoe", "Daniel Defoe", 252, true));
+    myLibrary.push(new Book("The Old Man and the Sea", "Ernest Hemingway", 127, false));
+  }
   render();
 });
 
-function populateStorage() {
-  if (myLibrary.length == 0) {
-    let book1 = new Book("Robison Crusoe", "Daniel Defoe", "252", true);
-    let book2 = new Book(
-      "The Old Man and the Sea",
-      "Ernest Hemingway",
-      "127",
-      true
-    );
-    myLibrary.push(book1);
-    myLibrary.push(book2);
-    render();
-  }
+function Book(title, author, pages, read) {
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+  this.read = read;
 }
 
-const title = document.getElementById("title");
-const author = document.getElementById("author");
-const pages = document.getElementById("pages");
-const check = document.getElementById("check");
-
-//check the right input from forms and if its ok -> add the new book (object in array)
-//via Book function and start render function
 function submit() {
-  if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
-  ) {
-    alert("Please fill all fields!");
-    return false;
-  } else {
-    let book = new Book(title.value, title.value, pages.value, check.checked);
-    library.push(book);
-    render();
+  const title = document.getElementById("title").value.trim();
+  const author = document.getElementById("author").value.trim();
+  const pages = parseInt(document.getElementById("pages").value);
+  const read = document.getElementById("check").checked;
+
+if (!title || !author || isNaN(pages) || pages <= 0) {
+    alert("Please fill in all fields correctly.");
+    return;
   }
+  
+   myLibrary.push(new Book(title, author, pages, read));
+  render();
 }
 
-function Book(title, author, pages, check) {
-  this.title = title;
-  this.author = author;
-  this.pages = pages;
-  this.check = check;
-}
+document.getElementById("title").value = "";
```

Comment:
If code is outside of a function, it will only run when the script first loads. 
For this application, it makes sense for the book form to be cleared after successful submission.

Can you review this code change, try it out locally and make the changes necessary to clear the form after a submit?